### PR TITLE
Dark mode support

### PR DESCRIPTION
**Pull request by henryiii**
Dark mode support (see #194).

You can see an example on the right here (slightly older version):

https://twitter.com/HenrySchreiner3/status/1174691026650521602/video/1

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -95,30 +117,27 @@ div.panel-body a.list-group-item.view-all {
...
-  color: white;
-  background: transparent;
-}
-img::-moz-selection {
+img::selection, img::-moz-selection {
   color: white;
   background: transparent;
 }
 
+
```
Comment (julian-alarcon) [inline static/css/main.css:130]: You can remove this @henryiii

**Comment by fubarhouse** [comment]
I've added a `custom.css` to implement this, thanks for your efforts @henryiii.
FWIW it works well, so if anybody wants testers you have my opinion.

**Comment by rterakedis** [comment]
I've also implemented these changes on my site with success:   https://blog.euc-rt.me

Thanks @henryiii

**Comment by halogenica** [comment]
Thank you very much @henryiii !

**Pull request merged**